### PR TITLE
First version of the Middleware

### PR DESCRIPTION
**Pull request by igui**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -77,11 +77,11 @@ Ready to contribute? Here's how to set up `django-api-query-count` for local dev
 5. When you're done making changes, check that your changes pass flake8 and the
    tests, including testing other Python versions with tox::
 
-        $ flake8 django_api_query_count tests
+        $ flake8 request_query_count tests
```
Comment (pricco) [inline CONTRIBUTING.rst]: It should be `test_query_count`, right?

File: request_query_count/query_count.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
+import json
+import re
+import sys
+from sys import maxsize, stderr
+
+ANY = '<any>'
+
+
+class QueryCountExclusion(object):
```
Comment (pricco) [inline request_query_count/query_count.py]: I think we should add a blank line between class or docstring and the next variable or function definition.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (igui) [reply CONTRIBUTING.rst]: Yes

File: request_query_count/query_count.py (hunk 2 above)
Comment (igui) [inline request_query_count/query_count.py]: I'm using flake8 (plus `cop.sh`),  and it complains if you don't put exactly 2 lines between first level definitions. In this case `ANY = ...` and class `QueryCountExclusion...`, so I think this is right

**Review by igui — COMMENTED** [review]
I will review the changes and push a new version

File: request_query_count/query_count.py (hunk 2 above)
Comment (igui) [reply request_query_count/query_count.py]: Sure, it can be a good idea

**Review by pricco — APPROVED** [review]

**Pull request merged**